src/status_im/contexts/wallet/wallet_connect/utils.cljs

```diff
@@ -1,10 +1,10 @@
 (ns status-im.contexts.wallet.wallet-connect.utils
   ;; NOTE: Not sorting namespaces since @walletconnect/react-native-compat should be the first
   #_{:clj-kondo/ignore [:unsorted-required-namespaces]}
-  (:require ["@walletconnect/react-native-compat"]
-            ["@walletconnect/core" :refer [Core]]
-            ["@walletconnect/web3wallet" :refer [Web3Wallet]]
+  (:require ["@walletconnect/core" :refer [Core]]
```

Comment:
probably this change should be reverted in light of the comment above : 

> ;; NOTE: Not sorting namespaces since @walletconnect/react-native-compat should be the first

Reply:
Aha, this is being done by the linter.
cc @ilmotta, @clauxx  @briansztamfater - do we have a way to stop sorting automatically on some files? 🤔

Reply:
We do @J-Son89, but only if we stop linting the entire file. Check the line calling `clojure-lsp <flags> clean-ns` command in the `lint-fix` target. You can add another file there to be ignored.

Reply:
@ajayesivan